for_workspace/utils_osx.sh

```diff
@@ -49,7 +49,8 @@ function path() {
 # $3 replace target
 function replace_in_files() {
   if [ -d "$1" ]; then
-    find -L $1 -type f \( -name "*.pc" -or -name "*.la" \) -exec sed -i 's@'"$2"'@'"$3"'@g' {} ';'
+# !!! the command line find and sed options are slightly different from Linux
```

Comment:
Of course, but I would keep it for now as these files continue to evolve. I had some ideas of maybe keeping only the functions that differ in osx and win files, and just load unix + osx/win in the script. That would allow better understanding.